smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/EmitterConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package io.smallrye.reactive.messaging.extension;
+
+import org.eclipse.microprofile.reactive.messaging.OnOverflow;
+
+import io.smallrye.reactive.messaging.annotations.Broadcast;
+
+public class EmitterConfiguration {
+    private final String name;
+    private OnOverflow.Strategy overflowBufferStrategy = null;
```

Comment:
Should we make all the fields final and public?

That would avoid getters, and be "like a record".

Reply:
It would mean the constructor setting values to null based on what's based in vs setting the default on the field itself.

Is that ok?

Reply:
The constructor would have to set the fields to `null` / `false`/ `-1`. 
I don't have a strong opinion about it, I'm not sure it simplifies much.

Reply:
So... Leave or change? I don't mind

Reply:
I will include the change for now